org.eclipse.draw2d/src/org/eclipse/draw2d/BasicColorProvider.java

```diff
@@ -0,0 +1,128 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *     Georgii Gvinepadze - georgii.gvinepadze@dbeaver.com
+ *     Serge Rider - serge@dbeaver.com
```

Comment:
In the other eclipse projects we do for first files:

  Georgii Gvinepadz, Serge Rider - initial API and implementation